htpasswd.go

```diff
@@ -118,7 +118,12 @@ func (h *HtpasswdFile) Reload() error {
 		return err
 	}
 	users := make(map[string]string)
+	valid, _ := regexp.Compile(`^[\p{L}.-]+$`)
```

Comment:
Here the error is ignore. The common thing to do is to have a top-level variable (which is only initialized once) like this:
```go
var validUsernameRegexp = regepx.MustCompile(`^[\p{L}.-]+$`)
```

`MustCompile()` panics if the regular expression is invalid.

Reply:
Thanks :+1: